doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/05_Pimcore_Bundles/README.md

```diff
@@ -9,29 +9,23 @@ in order to show up in the `pimcore:bundle:list` command. This gives you the fol
 * The bundle adds methods to natively register JS and CSS files to be loaded with the admin interface and in editmode.
 
 To get started quickly, you can extend `Pimcore\Extension\Bundle\AbstractPimcoreBundle` which already implements all methods
-defined by the interface. Besides name, description and version as shown in the extension manager, the `PimcoreBundleInterface` interface defines the following methods you
-can use to configure your bundle:
+defined by the interface. Besides name, description and version as shown in the extension manager, the `PimcoreBundleInterface`
+interface defines the following methods you can use to configure your bundle:
 
 ```php
-interface PimcoreBundleInterface extends BundleInterface
+interface PimcoreBundleInterface extends AbstractPimcoreBundle implements PimcoreBundleInterface
```

Comment:
Thank you for the PR!

Beside that it should go on a bugfix branch instead of 11.x, i have just noticed an error here, it's implementing itself
interface `PimcoreBundleInterface` extends AbstractPimcoreBundle implements `PimcoreBundleInterface`
https://github.com/pimcore/pimcore/blob/1f4646754543085a14560f1f884dfb0183f79780/lib/Extension/Bundle/AbstractPimcoreBundle.php#L21

but i am not getting the change, looks fine, if the purpose of the doc was to extract the current code
https://github.com/pimcore/pimcore/blob/1f4646754543085a14560f1f884dfb0183f79780/lib/Extension/Bundle/PimcoreBundleInterface.php#L22

Reply:
Probably would be easier and safer to just have a link to the PimcoreBundleInterface in here.

Doesn't seem necessary that it cites parts of the source code

Reply:
I just took over the branch due inactivity and merged.
Thank you again for reporting!